test/factbase/indexed/test_indexed_or.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+# SPDX-FileCopyrightText: Copyright (c) 2024-2025 Yegor Bugayenko
+# SPDX-License-Identifier: MIT
+
+require_relative '../../test__helper'
+require_relative '../../../lib/factbase'
+require_relative '../../../lib/factbase/term'
+require_relative '../../../lib/factbase/taped'
+require_relative '../../../lib/factbase/indexed/indexed_term'
+require_relative '../../../lib/factbase/indexed/indexed_or'
+
+# Indexed term 'or' test.
+# Author:: Yegor Bugayenko (yegor256@gmail.com)
+# Copyright:: Copyright (c) 2024-2025 Yegor Bugayenko
+# License:: MIT
+class TestIndexedOr < Factbase::Test
+  def test_predicts_on_or
+    term = Factbase::Term.new(
+      :or,
+      [
+        Factbase::Term.new(:exists, [:bar]),
+        Factbase::Term.new(:eq, [:foo, 42]),
+        Factbase::Term.new(:eq, [:bar, 7])
+      ]
+    )
+    idx = {}
+    term.redress!(Factbase::IndexedTerm, idx:)
+    maps = Factbase::Taped.new([{ 'foo' => [42] }, { 'bar' => [7], 'foo' => [22, 42] }, { 'foo' => [22, 42] }])
+    n = term.predict(maps, nil, {})
+    assert_equal(3, n.size)
+    assert_kind_of(Factbase::Taped, n)
+  end
+
+  def test_predicts_on_or_with_nil
```

Comment:
@maxonfjvipon maybe you can add more tests? Now this PR looks like a very trivial change.